src/ansible_compat/runtime.py

```diff
@@ -77,6 +83,29 @@ def __init__(
             raise RuntimeError(
                 f"Found incompatible version of ansible runtime {self.version}, instead of {min_required_version} or newer."
             )
+        if require_module:
+            self._check_module()
+
+    def _check_module(self) -> None:
+        """Assure that Ansible Python module is installed and matching CLI version."""
+        ansible_module_version = None
+        try:
+            # pylint: disable=import-outside-toplevel
+            from ansible.release import __version__
+
+            ansible_module_version = packaging.version.parse(__version__)
+
+        except (ImportError, ModuleNotFoundError):
```

Comment:
`ModuleNotFoundError` is a subclass of `ImportError`. There's no point in listing it additionally — it'll never be hit because `ImportError` is first in the tuple and will be matched first:
```suggestion
        except ImportError:
```